problems/33.Is Subsequence.gs

```diff
@@ -1,33 +1,15 @@
 function isSubsequence(s: string, t: string): boolean {
+    let j = 0
 
-    let subS: string[] = s.split("")
-    let str: string[] = t.split("")
-    let sumOfI: number[] = []
-    let k = 0
-
-    for (let i = 0; i < subS.length; i++) {
-        for (let j = 0; j < str.length; j++) {
-            if (subS[i] == str[j]) {
-                if (i > j) {
-                    continue
-                }
-                sumOfI[k] = i + j
-                k++
-                str[j] = "0"
-                break
-            }
+    for (let i = 0; i < t.length; i++) {
+        if (t[i] == s[j]) {
+            j++
```

Comment:
You have one special case here you didn't cover. What if t = 'abbbbb' s='ab' what will happen with j exactly? What will you test - be precise thing about it...